### DISCUSSION TITLE
Can I set the selected tab with code?

### DISCUSSION THREAD

**Discussion by rulai-yinqiang** [harborsiem/WindowsRibbon#25] [Q&A] [answered]
If I have 2 tabs in Ribbon, can I set the 2nd tab as the default selected tab in form_load ?

**Comment by harborsiem** [comment]
Nothing found if this is possible.

**Comment by harborsiem** [comment, marked as answer]
I have done some tests with Keytips on the tabs. In form shown event I do a key simulation with native function SendInput. This will set 2nd tab as default. See project:

[TestRibbonKeytip.zip](https://github.com/harborsiem/WindowsRibbon/files/15246076/TestRibbonKeytip.zip)

**Comment by rulai-yinqiang** [comment]
That's really a good idea!
